package.json

```diff
@@ -48,10 +50,6 @@
     "unused": true
   },
   "dependencies": {
-    "google-closure-library": "^20180204.0.0",
-    "gulp": "^3.9.1",
-    "install": "^0.8.8",
```

Comment:
I think this line was mistakenly added by an accidental `npm install install` command.